cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -313,68 +332,21 @@ function run_validation() {
   fi
 }
 
-function configure_hive_warehouse_dir() {
-  # Wait for master 0 to create the metastore db if necessary.
-  run_with_retries run_validation
-
-  local hiveserver_uri
-  hiveserver_uri=$(get_hiveserver_uri)
-  HIVE_WAREHOURSE_URI=$(beeline -u "${hiveserver_uri}" -e "describe database default;" |
-    sed '4q;d' | cut -d "|" -f4 | tr -d '[:space:]')
-
-  echo "Hive warehouse uri: $HIVE_WAREHOURSE_URI"
-
-  bdconfig set_property \
-    --name 'hive.metastore.warehouse.dir' \
```

Comment:
It seems that now this property never gets set for any of the 3 masters. Is that okay? It seems like it might be okay because it looks like previously, it was being set after we restarted the hive-metastore service anyways, so maybe it wasn't actually being used?

Reply:
If we do need to keep setting this property, we could extract the warehouse uri via mysql CLI like:
`
 HIVE_WAREHOURSE_URI=$(mysql -u "${db_hive_user}" "${db_hive_password_parameter}" -Nse \	    mysql -u "${DB_ADMIN_USER}" "${DB_ADMIN_PASSWORD_PARAMETER}" -e \
      "SELECT DB_LOCATION_URI FROM ${metastore_db}.DBS WHERE NAME = 'default';")
`

You can pass `--host` and `--port` to mysql command to make mysql point at cloud sql proxy on master 0, but will need to do this with retries on master 1 and 2 in case master 0 isn't ready yet

Reply:
Yes, I think this is how it should be - users needs to configure it explicitly or Hive will take it from DB automatically.